src/gui/component.container.js

```diff
@@ -1,6 +1,6 @@
 import { connect } from 'react-redux'
 import { getComponentProps } from './redux/model'
-import Component from './component.js'
+import Component from './component.jsx'
```

Comment:
ok, I see next file, what about renaming `component.jsx` to `component.loaded.js(x)` ?